### PR TITLE
Common properties

### PR DESCRIPTION
**Pull request by AlFasGD**
This adds the ability to retrieve the common properties of a level object collection, including some changes to the property access info system, which will be reworked again.

### CODE REVIEW

File: GDAPI/GDAPI/Utilities/Objects/General/ListList.cs [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+
+namespace GDAPI.Utilities.Objects.General
+{
+    /// <summary>Represents a list of lists of objects of type <seealso cref="T"/>.</summary>
+    /// <typeparam name="T">The type of the objects to store in the lists.</typeparam>
+    public class ListList<T> : IEnumerable<T>
```
Comment (Altenhh) [inline GDAPI/GDAPI/Utilities/Objects/General/ListList.cs]: NestedLists?

File: GDAPI/GDAPI/Utilities/Enumerations/GeometryDash/ObjectParameter.cs [hunk 2]
```diff
@@ -10,6 +10,7 @@
 
 namespace GDAPI.Utilities.Enumerations.GeometryDash
 {
+    // TODO: Rename to ObjectProperty to consistenize terminology
```
Comment (Altenhh) [inline GDAPI/GDAPI/Utilities/Enumerations/GeometryDash/ObjectParameter.cs]: You should clear this TODO

File: GDAPI/GDAPI/Utilities/Objects/General/ListList.cs (hunk 1 above)
Comment (AlFasGD) [reply GDAPI/GDAPI/Utilities/Objects/General/ListList.cs]: Looks better

File: GDAPI/GDAPI/Utilities/Enumerations/GeometryDash/ObjectParameter.cs (hunk 2 above)
Comment (AlFasGD) [reply GDAPI/GDAPI/Utilities/Enumerations/GeometryDash/ObjectParameter.cs]: This is for another PR where terminology will be broadly consistenized.

File: GDAPI/GDAPI/Utilities/Objects/General/ListList.cs (hunk 1 above)
Comment (Altenhh) [reply GDAPI/GDAPI/Utilities/Objects/General/ListList.cs]: Yes, but it doesn't read very well

File: GDAPI/GDAPI/Utilities/Enumerations/GeometryDash/ObjectParameter.cs (hunk 2 above)
Comment (Altenhh) [reply GDAPI/GDAPI/Utilities/Enumerations/GeometryDash/ObjectParameter.cs]: ah, ok

**Pull request merged**